packages/hoppscotch-app/src/components/smart/EnvInput.vue

```diff
@@ -42,6 +42,7 @@ const props = withDefaults(
     styles?: string
     envs?: { key: string; value: string; source: string }[] | null
     focus?: boolean
+    isTextSelected?: boolean
```

Comment:
I am not a fan of using props with this naming scheme to define state-y operations.

Props are philosophically meant to be representative of the rendering state by name association but this doesn't align well with that concept.

Either, rename the props to express these actions are done on mount (`focusOnMount`, `selectTextOnMount` etc.) or move the move and focus options to component methods and expose it so the parent can call it via a template ref.

I know `focus` is from existing code but it was not being used even apparently so it slipped my radar I guess (@anwarulislam)